app/coffeescripts/views/conferences/ConferenceView.coffee

```diff
@@ -20,9 +20,10 @@ define [
   'jquery'
```

Comment:
Do we not have any javascript specs around these ConferenceView changes? Would be nice to get some unit tests here, and if those new unit tests cover what is being currently tested by some of the selenium specs, remove those selenium specs.

Reply:
Nope, there were no tests at all on conferences. I agree that more should be added. Although in this case I focused all the effort only in the new feature for big_blue_button as the Conference spec is actually testing Wimba (only).

I would suggest to complete this one and we can certainly help adding some more tests in a next PR if you wanted to.